extensions/typora-note-creator/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "typora-note-creator",
+  "title": "Typora Note Creator",
+  "description": "This extension allows you to create folders and notes for Typora from templates.",
+  "icon": "extension_icon.png",
+  "author": "mynameisny",
+  "license": "MIT",
+  "categories": ["Productivity"],
+  "commands": [
+    {
+      "name": "create-note",
+      "title": "Create Note",
+      "description": "Create a new note in Typora",
+      "mode": "view",
+      "shortcut": {
+        "key": "n",
+        "modifiers": ["cmd", "shift"]
+      }
+    }
+  ],
+  "preferences": [
+    {
+      "name": "notesDir",
+      "title": "Notes Directory",
+      "description": "The directory where notes are stored",
+      "type": "directory",
+      "required": true,
+      "placeholder": "Select the directory to store notes",
+      "default": "~/Documents/Notes"
```

Comment:
Maybe not set a default since not many has that directory?

Reply:
I've removed the default notes directory. Please review again when convenient. Thanks!